### DISCUSSION TITLE
How do we update supabase to the latest version?

### DISCUSSION THREAD

**Discussion by htetlynnhtun** [coollabsio/coolify#4509] [Q&A]
Recently, Supabase team release a couple new features like integration, queue, cron job. I want to use them on my instance deployed by coolify. How can I update supabase in coolify?

**Comment by skusez** [comment]
you need to update the image to the latest version on every service. Find this in settings
![Screenshot 2024-12-10 at 10 44 35 am](https://github.com/user-attachments/assets/465bc72f-0853-478c-95fe-3197a3d54ace)

![Screenshot 2024-12-10 at 10 45 32 am](https://github.com/user-attachments/assets/b7d4cab6-c570-4d61-985a-a6ca563dc0ae)

From there you can follow this guide to find the right image
https://supabase.com/docs/guides/self-hosting/docker#updating-your-services

I dont all the features are released for self hosted yet, I'm eagerly waiting for them to drop as well
